### PR TITLE
feat: add docker setup script

### PR DESCRIPTION
**Pull request by ricetech**

### CODE REVIEW

**Review by ajayyy — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -104,6 +104,20 @@ Further information about how the scraper works can be found in [it's dedicated
...
 
-Then you can run `docker-compose up` to start the application.
\ No newline at end of file
+```bash
+cd docker
+./setup.sh
+```
+
+Then, fill in the values in each of the following `*.env` files:
+- `database.env`
```
Comment (seendsouza) [inline README.md:115]: Consider starting env files with . so they are hidden

File: README.md [hunk 2]
```diff
@@ -104,6 +104,20 @@ Further information about how the scraper works can be found in [it's dedicated
...
+- `server.env`
+- `caddy.env`
+
+The other `.env` files have default values, or are blank. They must exist but do not need to be edited.
+
+**WARNING: The credentials in `docker/database.env` must match the credentials in the `DATABASE_URL` in `docker/server.env`.**
+
+Then you can run `docker-compose up` to start the application.
```
Comment (seendsouza) [inline README.md:123]: docker compose up bc docker-compose sometimes doesn't work on certain platforms

Comment (ricetech) [reply README.md]: Resolved in https://github.com/6monthslater/6monthslater/pull/97/commits/4f2ad5ee9b8c4c556391b1f82001913d6a24e1ac

**Review by seendsouza — APPROVED** [review]
lgtm

**Pull request merged**